docs/topics/secured-mission-common-interaction.adoc

```diff
@@ -45,3 +115,21 @@ NOTE: This is the master realm public key from the Secured booster's {RHSSO} ser
 
 . Paste the `token` output from the client output in the _Encoded_ box.
 . You will see `Signature Verified` on the debugger page.
+
+== Using the Web Interface
+The secured endpoint also contains a web interface to interact with it. Access it using the xref:secured-endpoint[endpoint address].
+
+When using the web interface, you will go through the following steps to see how security is enforced, how authentication happens, and how the token is requested and attached to an authenticated invocation.
+
+. Open the web interface and click on _Invoke_. This sends an unauthenticated request to the Greeting service. 
+. The services responds with an `HTTP 401 Unauthenticated` status code.
+. Click _Login_ to authenticate yourself against {RHSSO}. This redirects you to the SSO server and requests your credentials. 
+. Log in as xref:sso-users[the _Alice_ user].
+. Once authenticated you are redirected to the web interface, which includes a bearer token shown in the command line output at the bottom of the page.
+. Click _Invoke_ again to access the Greeting service.
+. Confirm there is no exception and JSON response payload displayed. This means the service accepted your bearer token and you are authorized access to the Greeting service.
+. Logout.
+. Click on _Invoke_. Confirm this sends an unauthenticated request to the Greeting service. 
+. Click _Login_ and log in as xref:sso-users[the _admin_ user].
+. Click on _Invoke_. 
+. Confirm the service responds with an `HTTP 403 Unauthorized` status code since the _admin_ user is not authorized to access the Greeting service.
```

Comment:
One more issue here. previously, the 1st item in thist list had `401 Unauthorized`, and I asked you to add `403 Forbidden` to this last item of the list. What you did was changing `401 Unauthorized` to `401 Unauthenticated` and added `403 Unauthorized`.

It's true that 401 meands unauthenticated and 403 means unauthorized, but the typical status messages are `401 Unauthorized` and `403 Forbidden`. Could you please change that back? Thanks.

Reply:
sorry, missed this when merging. i'll file a follow up PR @Ladicek

Reply:
Sure, np.